src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -99,4 +107,17 @@ public static function checkForIllegalCsvPropertyKeys($propertyKey)
             throw new BadPropertyKeyException($propertyKey);
         }
     }
+
+    /**
+     * Verifies that the provided value is not a object.
+     *
+     * @param mixed $value The value to check.
+     * @throw ObjectNotAllowedException In case the provided value is a object.
+     */
+    public static function checkForObjectType($value)
```

Comment:
Given that we change the check to check for permitted values instead of restricted ones, we should probably better call it something like `checkForAcceptableType` or just `checkType`.